plugins/hls-cabal-plugin/src/Ide/Plugin/Cabal/Completion/CabalFields.hs

```diff
@@ -46,6 +50,71 @@ findFieldSection cursor (x:y:ys)
   where
     cursorLine = Syntax.positionRow cursor
 
+-- | Determine the field line the cursor is currently a part of.
+--
+-- The result is said field line and its starting position
+-- or Nothing if the passed list of fields is empty.
+
+-- This function assumes that elements in a field's @FieldLine@ list
+-- do not share the same row.
```

Comment:
Do I get this right, the assumption is that each `FieldLine` is actually on a unique line? Thanks for documenting this assumption, and I am sure, this assumption holds :)